src/resolvers/cross-schema/index.ts

```diff
@@ -10,6 +10,7 @@ export const crossSchemaExtensions = `
   extend type Contract {
     perils(locale: Locale!): [PerilV2!]!
     insurableLimits(locale: Locale!): [InsurableLimit!]!
+    termsAndConditions(locale: Locale!): InsuranceTerm!
```

Comment:
From my understanding this field is now deprecated let's then be explicit
```suggestion
   @deprecated(reason: "Use `termsAndConditions` form `AgreementCore` instead")
    termsAndConditions(locale: Locale!): InsuranceTerm!
```